src/test/java/io/nats/client/impl/JetStreamManagementTests.java

```diff
@@ -1305,6 +1306,7 @@ private void validateGetMessage(JetStreamManagement jsm, TestingStreamContainer
 
         assertStatus(10003, assertThrows(JetStreamApiException.class, () -> jsm.getMessage(tsc.stream, -1)));
         assertStatus(10003, assertThrows(JetStreamApiException.class, () -> jsm.getMessage(tsc.stream, 0)));
+        assertStatus(10003, assertThrows(JetStreamApiException.class, () -> jsm.getFirstMessage(tsc.stream, DEFAULT_TIME)));
```

Comment:
Just curious, is it correct that this fails because the start time is before the very first message in the stream?

Reply:
This doesn't fail because of the start time being before the first message.
Instead it fails because a `DEFAULT_TIME` doesn't get serialized into the final JSON, resulting in an empty JSON object being sent which is checked here should be an error.